torch/_dynamo/variables/torch.py

```diff
@@ -288,6 +288,8 @@ def call_function(
         ):
             log.warning("Profiler will be ignored")
             return ProfilerContextWrapperVariable(**options)
+        elif self.value is torch.autograd._profiler_enabled:
+            unimplemented("torch.autograd._profiler_enabled not supported yet")
```

Comment:
Do we really want to graph break? Please check L289, actually we ignore all ```torch.profiler.*``` ops, because of [this](https://github.com/pytorch/torchdynamo/pull/867#issuecomment-1223386064). So why don't we just return the value and print out warning like L289? I think we are always trying to avoid graph break if we can.

Reply:
Thank you for the link to the comment! Yes, we have definitely noticed L289. As we currently just ignore profiler and we do not expect different numerical behavior with profiler or not, we can trace it as a constant. @voznesenskym , what do you think?

Reply:
ProfilerContextWrapperVariable is better than a constant